src/test/bufferlist.cc

```diff
@@ -2369,6 +2369,20 @@ TEST(BufferList, splice) {
     EXPECT_EQ((unsigned)11, bl.length());
     EXPECT_EQ(0, ::memcmp("fghi", other.c_str(), other.length()));
   }
+
+  {
+    bufferlist bl1, bl;
```

Comment:
It would be helpful to have a comment here explaining the block is actually a bug replicator. Also, there is no any `EXPECT_...` clause.

Reply:
> It would be helpful to have a comment here explaining the block is actually a bug replicator. Also, there is no any `EXPECT_...` clause.

done, please check again.